mvpa2/base/constraints.py

```diff
@@ -10,16 +10,20 @@
 
 # TODO: __str__ / __repr__'s
 
+from builtins import map
+from builtins import str
+from past.builtins import basestring
+from builtins import object
```

Comment:
why explicit imports `from builtins` are necessary? shouldn't it all work just fine without?